scripts/socks/README.md

```diff
@@ -40,7 +42,6 @@ $ SOCKS_NETWORK="USB 10/100/1000 LAN" ./socks.sh off
 
 The socks script will start the socks proxy, configure the proxy settings on your computer to use a `proxy.pac` file, and start a small webserver to serve that file up. The proxy.pac file will send only a few URLs to the proxy, sending everything else directly to the internet.
 
-Finally, you'll need to configure your browser to use the system proxy if you use this script. This has been tested on OSX 10.14
-and 10.15, and works with Safari, Firefox, and Chrome.
+Finally, If you want to use a browser other than Chrome or Safari, you'll need to configure your browser to use the system proxy. This has been tested on OSX 10.14 and 10.15, and works with Chrome, Firefox and Safari. Chrome and Safari use the system proxy by default; you can set it up for Firefox by following [these instructions](https://support.mozilla.org/en-US/kb/connection-settings-firefox).
```

Comment:
I didn't have to configure Firefox in a special way. It just worked like Chrome and Safari.

Reply:
Hmm - ok that's interesting. This may vary based on machine, version of firefox, or other system settings. I think since Firefox (of the 3) is the only one that actually has it's own proxy settings in the control panel, it's worth having the link there nonetheless